examples/build.gradle

```diff
@@ -132,11 +132,18 @@ subprojects {
             println "Launched Kernel Server 4. !!!"
         }
     }
-    task runenvapp(type: ExecWait, dependsOn: ksTask) {
-        command './gradlew ' + project + ':runapp'
+    task runenvapp(dependsOn: ksTask) {
+        // If any failures in 'runapp' task then 'runenvapp' task also should fail.
+        finalizedBy 'runapp'
+        // 'cleanup' task should run after runapp task always. Currently it is running after 'runapp' task.
+        // Dont move this task before 'runapp' task.
+        finalizedBy 'cleanup'
     }
 
     task run (dependsOn: runenvapp) {
+    }
+
+    task cleanup (dependsOn: runenvapp) {
```

Comment:
IMO, Cleanup task need not depend on ```runenvapp``` task

Reply:
done